README.md

```diff
@@ -1,8 +1,62 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Marissa Schmitt . Jarvis Consulting
 
+I am a recent graduate from Conestoga College where I studied Software Engineering Technology. I am excited to be challenged by new technologies and expand my skillset as I have learned a variety of programming languages during the duration of my program. During my time at Conestoga College, I learned about web development, mobile development, relational databases, object-oriented programming, and assembly language. My interest in software engineering started in high school where I pursued many computer science and computer engineering courses. I am intrigued by all aspects of software engineering whether it be frontend, backend, or working with a database directly. Currently I am working as a software developer/data engineer at Jarvis to improve my skills by learning about core Java programming, design patterns, Maven, and agile development.
```

Comment:
I am a recent graduate from Conestoga College, where I studied Software Engineering Technology. I am excited to be challenged by new technologies and expand my skill set as I have learned various programming languages during the duration of my program. During my time at Conestoga College, I have gained experience in web development, mobile development, RDBMS, OOP, and assembly language. My interest in software engineering started in high school, where I pursued many computer science and computer engineering courses. I am intrigued by all aspects of software engineering, whether front-end, back-end, or directly working with a database. Currently, I am working as a software developer and data engineer at Jarvis to improve my skills by learning the latest technologies.